### DISCUSSION TITLE
Appearance / Shape Appearance do nothing

### DISCUSSION THREAD

**Discussion by 0penBrain** [FreeCAD/FreeCAD#14457] [Code contribution discussions]
Hi all,
Just wondering if this is under control that `Appearance` task -> `Shape Appearance` is completely broken and actually do nothing at all.
All changed values are discarded. So there is no way to change the shape color except going to the Property Pane.
![image](https://github.com/FreeCAD/FreeCAD/assets/48731257/bb184480-ea4c-41a9-a67a-3a2b33c2849f)

@davesrocketshop

Thanks

**Comment by davesrocketshop** [comment]
There was a merge yesterday that addresses this https://github.com/FreeCAD/FreeCAD/pull/13792
